Sensus.UI.Tests/Tools/Tests.cs

```diff
@@ -1,175 +1,178 @@
+        #region Old Tests
+        //[Test, Ignore("Doesn't work. Old Test.")]
+        //public void ReadEvaluatePrintLoop()
+        //{
+        //    _app.Repl();
+        //}
+
+        //[Test, Ignore("Doesn't work. Old Test.")]
+        //public void RunUnitTestingProtocol()
+        //{
+        //    // wait a bit for app to start up -- ios sometimes displays permissions dialogs that need time to be dismissed (UiTest appears to take care of these on its own)
+        //    Thread.Sleep(5000);
+
+        //    // protocol has been started and is waiting for user consent
+        //    ConsentToProtocolStart(new TimeSpan(0, 0, 5));
+
+        //    // stop and edit protocol
+        //    StopProtocol();
+        //    TapProtocol(false);
+        //    _app.WaitForElementThenTap(PROTOCOL_ACTION_SHEET_EDIT, false);
+
+        //    // set data store delays such that they will be testable within a reasonable time period
+        //    TimeSpan localDataStoreDelay = new TimeSpan(0, 0, 5);
+        //    _app.WaitForElementThenTap(LOCAL_DATA_STORE_EDIT, true);
+        //    _app.WaitForElementThenEnterText(DATA_STORE_COMMIT_DELAY, true, localDataStoreDelay.TotalMilliseconds.ToString());
+        //    _app.WaitForElementThenTap(DATA_STORE_OK, true);
+
+        //    TimeSpan remoteDataStoreDelay = new TimeSpan(0, 0, 15);
+        //    _app.WaitForElementThenTap(REMOTE_DATA_STORE_EDIT, true);
+        //    _app.WaitForElementThenEnterText(DATA_STORE_COMMIT_DELAY, true, remoteDataStoreDelay.TotalMilliseconds.ToString());
+        //    _app.WaitForElementThenTap(DATA_STORE_OK, true);  // to protocol page
+        //    _app.Back();  // to protocols page
+
+        //    // restart protocol, wait for remote data store to commit data, and then check status
+        //    StartProtocol(new TimeSpan(0, 0, 5));
+        //    Thread.Sleep(remoteDataStoreDelay.Add(new TimeSpan(0, 0, 5)));
+        //    AssertProtocolStatusEmpty("Protocol status after remote data store.");
+
+        //    // enable accelerometer to check script triggering
+        //    TapProtocol(true);
+        //    _app.WaitForElementThenTap(PROTOCOL_ACTION_SHEET_EDIT, false);
+        //    _app.WaitForElementThenTap(PROBES_EDIT, true);
+        //    _app.WaitForElementThenTap(ACCELEROMETER_PROBE, true);
+        //    _app.WaitForElementThenTap(ACCELEROMETER_ENABLED, true);
+        //    Thread.Sleep(10000);  // 5-second accelerometer stabilization + a few seconds for the prompt page to show up before trying to scroll to it.
+        //    _app.WaitForElementThenEnterText(ACCELEROMETER_TRIGGERED_SCRIPT_INPUT, true, "12345");
+        //    _app.WaitForElementThenTap(PROMPT_FOR_INPUTS_SUBMIT, true);
+        //    _app.WaitForElementThenTap(PROMPT_FOR_INPUTS_SUBMIT_CONFIRM, false);
+        //    _app.Back();  // to probes page
+        //    _app.Back();  // to protocol page
+        //    _app.Back();  // to protocols page
+
+        //    StopProtocol();
+        //}

```

Comment:
Yeah, that's why I commented it out instead of removing it. None of my stuff runs on XTC.  However, when running my tests these get pulled in too, since they are also unit tests.

My argument against these tests would go like this. Xamarin promotes its UITest framework as unit tests but I think it is more of an integration testing tool. In my experience the only teams that perform automated integration testing are those with full time developers on the QA team. Otherwise it is simply too costly. Integration tests tend to be very brittle and therefore take lots of time to maintain.

My opinion is that we leave these commented out until we decide to spend the time to fix them.